source/default/_patterns/00-protons/index.js

```diff
@@ -1,9 +1,32 @@
 /**
- * Base css generation and global js logic
+ * Base css generation and global js logic.
  */
 
 import './_base.scss';
 
+import { mediaBreakpoint } from 'protons/utilities';
+import scssVariables from '../../_data/scssVariables.json';
+
+import enquire from '../../../node_modules/enquire.js';
```

Comment:
Should just be able to `import enquire from 'enquire.js;`. Let me know if the linter gets pissy about the `.js` on the end of a node_module name.

Reply:
@illepic, it's not the linter, I think it's whatever magical namespacing service we use to pull in node_modules. It doesn't get it. I actually used to have a comment here saying as much; my bad for removing it.

Reply:
All node modules are available by name, no need for `../../../`.

Reply:
@illepic, this node_module does not load by name. I can't tell you why. Ping me if you want proof.

Reply:
![image](https://user-images.githubusercontent.com/295127/47757012-9be1b900-dc61-11e8-81be-3e3c6db72110.png)
![image](https://user-images.githubusercontent.com/295127/47757024-b025b600-dc61-11e8-8896-40be594153d2.png)